contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -1,68 +1,704 @@
+          (NXpid):
+            exists: recommended
+            setpoint(NX_FLOAT):
+              exists: recommended
+        device_information(NXfabrication):
+          exists: recommended
+          vendor:
+            exists: recommended
+          model:
+            exists: recommended
+          identifier:
+      pressure_gauge(NXsensor):
+        exists: recommended
+        doc: |
+          Device to measure the gas pressure around the sample.
+        name:
+          exists: recommended
+        measurement:
+          enumeration: [pressure]
+        type:
+          exists: optional
+        value(NX_FLOAT):
+          unit: NX_PRESSURE
+          doc: |
+            In case of a single or averaged gas pressure measurement, this is the scalar gas pressure around
+            the sample. It can also be an 1D array of measured pressures (without time stamps).
+        value_log(NXlog):
+          exists: optional
+          value(NX_NUMBER):
+            unit: NX_PRESSURE
+            doc: |
+              In the case of an experiment in which the gas pressure changes and is recorded,
+              this is an array of length m of gas pressures.
+      flood_gun(NXactuator):
+        exists: optional
+        doc: |
+          Device to bring low-energy electrons to the sample for charge neutralization
+        name:
+          exists: recommended
+        physical_quantity:
+          enumeration: [current]
+        type:
+          exists: optional
+        current(NX_FLOAT):
+          exists: recommended
+          unit: NX_CURRENT
+          doc: |
+            In case of a fixed or averaged electron current, this is the scalar current.
+            It can also be an 1D array of output current (without time stamps).
+        current_log(NXlog):
+          exists: optional
+          value(NX_NUMBER):
+            unit: NX_CURRENT
+            doc: |
+              In the case of an experiment in which the electron current is changed and
+              recorded with time stamps, this is an array of length m of current setpoints.
+      history(NXhistory):
+        exists: optional
+        doc: |
+          A set of activities that occurred to the instrument prior to/during the photoemission
+          experiment, including any activities performed on the individual instrument parts.
+          This group can be used to describe the preparation of the instrument prior to the
+          experiment, e.g. the cleaning procedure for a spin filter crystal.
+    (NXprocess_mpes):
+      exists: recommended
+      doc: |
+        Document an event of data processing, reconstruction, or analysis for this data.
+        The appropriate axis calibrations for a given experiment should be described using
+        one or more of the following NXcalibrations. The individual calibrations for a given
+        `AXISNAME` in `data` should be called `AXISNAME_calibration`.
       energy_calibration(NXcalibration):
+        exists: optional
```

Comment:
I still don't like this very long list of options here in the appdef

Reply:
Basically, we are just saying here that the `calibrated_axis` objects are recommended (and the same for some other sub-elements of the different calibration processes). We can also remove these `exists`keywords and then there is no need to have this long list.